packages/cli-doctor/src/tools/healthchecks/dependencies.ts

```diff
@@ -0,0 +1,106 @@
+import fs from 'fs';
+import chalk from 'chalk';
+import path from 'path';
+import semver from 'semver';
+import {HealthCheckInterface} from '../../types';
+import {logManualInstallation} from './common';
+import {findProjectRoot} from '@react-native-community/cli-tools';
+
+const RNPackages = [
+  '@react-native/babel-plugin-codegen',
+  '@react-native/assets-registry',
+  '@react-native/eslint-config',
+  '@react-native/eslint-plugin-specs',
+  '@react-native/hermes-inspector-msggen',
+  '@react-native/metro-config',
+  '@react-native/normalize-colors',
+  '@react-native/js-polyfills',
+  '@react-native/bots',
+  '@react-native/codegen-typescript-test',
+  '@react-native/codegen',
+  '@react-native/gradle-plugin',
+  '@react-native/typescript-config',
+  '@react-native/virtualized-lists',
+];
+
+const getPackageJson = (root: string): Record<string, any> => {
+  try {
+    const packageJson = JSON.parse(
+      fs.readFileSync(path.join(root, 'package.json'), 'utf8'),
+    );
+    return packageJson;
+  } catch (e) {
+    return {};
+  }
+};
+
+const findDependencies = (root: string): Record<string, string> => {
+  const {devDependencies = {}, dependencies = {}} = getPackageJson(root);
+  return {
+    ...devDependencies,
+    ...dependencies,
+  };
+};
+
+export default {
+  label: 'Dependencies',
+  isRequired: false,
+  description: 'NPM dependencies needed for the project to work correctly',
+  getDiagnostics: async (_, config) => {
+    try {
+      const root = config?.root || findProjectRoot();
+      const dependencies = findDependencies(root);
+      const reactNativeVersion = dependencies['react-native'];
+      const reactNativeCoercedVersion = semver.coerce(reactNativeVersion);
+      const issues: string[] = [];
+
+      RNPackages.forEach((pkg) => {
+        if (dependencies[pkg]) {
+          const packageVersion = dependencies[pkg];
+          const packageCoercedVersion = semver.coerce(packageVersion);
+          if (reactNativeCoercedVersion && packageCoercedVersion) {
+            const verisonDiff = semver.diff(
+              packageCoercedVersion,
+              reactNativeCoercedVersion,
+            );
+            if (verisonDiff === 'major' || verisonDiff === 'minor') {
+              issues.push(
+                `   - ${chalk.red.bold(
```

Comment:
Let's keep the convention in whole `doctor` command and use here `chalk.red` instead of `chalk.red.bold`:
```suggestion
                `   - ${chalk.red(
```